pkg/mdformatter/linktransformer/link.go

```diff
@@ -551,7 +551,7 @@ func (l localLinksCache) addRelLinks(localLink string) error {
 func toHeaderID(header []byte) string {
 	var id []byte
 	// Remove punctuation from header except '-' or '#'.
-	// '\p{L}\p{N}\p{M}' is the unicode equivalent of '\w', https://www.regular-expressions.info/unicode.html.
+	// '\p{L}\p{N}\p{M}' is the Unicode equivalent of '\w', https://www.regular-expressions.info/unicode.html.
```

Comment:
```suggestion
	// '\p{L}\p{N}\p{M}' is the unicode equivalent of '\w', https://www.regular-expressions.info/unicode.html.
```

This one is correct honeslty

Reply:
but not a big deal